packages/common/src/initiatives/_internal/computeProps.ts

```diff
@@ -39,10 +39,7 @@ export function computeProps(
 
   initiative.view = {
     ...model.data.view,
-    featuredImageUrl: getAuthedImageUrl(
-      model.data.view.featuredImageUrl,
-      requestOptions
-    ),
+    featuredImageUrl: model.data.view.featuredImageUrl,
```

Comment:
Is removing this still necessary now that we're handing everything else with the featured images correctly?

Reply:
If you're going to remove this `getAuthedImageUrl`, I'm pretty sure you can just get rid of this entire thing, e.g. lines 40-43.